packages/db/src/migrations/0001_create_user_and_auth.js

```diff
@@ -5,7 +5,13 @@ import { createTableWithDefaults } from "../utils.js"
 export async function up(db) {
   const query = sql`CREATE EXTENSION IF NOT EXISTS ulid;`.compile(db)
   await db.executeQuery(query)
-  await createTableWithDefaults("ow_user", { id: true, createdAt: true }, db.schema).execute()
+  await createTableWithDefaults("ow_user", { id: true, createdAt: true, updatedAt: true }, db.schema)
+    .addColumn("email", "varchar(255)", (col) => col.notNull().unique())
+    .addColumn("given_name", "varchar(255)", (col) => col.notNull())
+    .addColumn("family_name", "varchar(255)", (col) => col.notNull())
+    .addColumn("name", "varchar(255)", (col) => col.notNull())
+    .addColumn("study_year", "integer", (col) => col.notNull().defaultTo(-1))
+    .execute()
```

Comment:
Good insights! I would like to discuss this more tomorrow, let's try not to forget

Reply:
Changing old migrations is a bad habit to acquire, and if you feel like monoweb is making it hard to properly make migrations we should actually do something about it. 

This change means that everyone that develops on monoweb has to recreate their database-schema and delete all their data which I think is unnecessary, even though it does not cause any harm to any production db this is like throwing a stick into the wheel of other developers

> This way it is clear what the user table includes, rather than having it in many different migration files.

Having fewer migration files is not a solution, people should not be reading migrations to figure out the current db schema

Reply:
I researched this more now and I realize that this change will indeed be very annoying for other people that are using their own database. 

Am I right in this thinking:

In this PR I am adding fields to the `ow_user` with the `not null` restriction. E.g. `name`.

The proper way of doing this would be to:
1. create new migration
2. create new fields WITHOUT not null restriction
3. modify all rows with values for the new fields that are not null
4. modify fields to have not null restriction

I have not been working with migrations before so for me this seems like a tideous process. However, maybe it's not, I don't know.

My workflow when creating these new migrations have been to just delete the database and create it before running migrations. But I understand this is not something people have set up.

Reply:
This is a pain to deal with if anything goes wrong, IMO devs _should just need to apply a migration_. It is valuable to have an examples of proper migration for later devs, and you benefit from learning how to do this properly. This kind of migration will be needed later as well.

I leave it to your discretion to find a suitable solution